### PR TITLE
Added a new page for language mismatch error troubleshooting

### PR DESCRIPTION
**Pull request by Jay-umbr**
The "Languages and source and destination sites do not match" exception does not come up often, but it is a bit of a headscratcher. This new page aims to help identify the possible issue, and provide steps on solving the error.

### CODE REVIEW

File: Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+versionFrom: 7.0.0
+---
+
+# Troubleshooting languages mismatch
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md:5]:
```suggestion
# Troubleshooting language mismatches
```

File: Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+If you are using dictionary items on a multi-lingual setup, you might see this error happen in some cases.
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md:7]:
```suggestion
If you are using dictionary items on a multi-lingual setup, you might see errors related to mismatches between languages.
```

File: Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Cause
+
+This error most often occurs when the default Umbraco language (English-US) is deleted from the backoffice. When creating a new site, or a new environment on a project, said project will always have the default English language added - which could cause your environments to become out of sync.
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md:11]:
```suggestion
This error most often occurs when the default Umbraco language (English-US) is deleted from the backoffice. When creating a new site, or a new environment on a project, said instance will always have the default English-US language added, which could cause your environments to become out of sync.
```

File: Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## Identify the issue
+
+The issue will show up as an extraction error on your Umbraco Cloud environment with a red indicator.
+
+![Extraction error](images/error.png)
+
+Upon closer inspection, a more detailed error will reveal itself: 'Languages in source and destination site do not match.'
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md:19]:
```suggestion
Upon closer inspection, a more detailed error will reveal itself: `Languages in source and destination site do not match.`
```

File: Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+This error can occur in two scenarios, and is caused by deleting the default 'English (United States)' backoffice language, while having **dictionary items** present on your project.
+
+If the below conditions are met, you will be presented with the 'Languages in source and destination site do not match.' error:
+
+1. You have at least two languages set in the backoffice
+
+2. The default English-US language is not present amongst your languages
+
+3. Your document types are set to "Vary by language"
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md:31]:
```suggestion
3. Your Document Types are set to "Vary by language"
```

**Comment by Jay-umbr** [comment]
I need to take a look at this again - it seems the only culprit is in fact the dictionary items.
Will update the thread once I am done

**Comment by Jay-umbr** [comment]
Corrected the article

File: Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+The second scenario (*Scenario 2*) is when you create a new project from a baseline, where the baseline would be your project with dictionary items.
+
+## Fixing
+
+Thankfully, this issue is relatively easy to fix - and there are 2 ways to get this error resolved.
+
+(*Method 1*) - prevention is better than the cure! If you resave all your dictionary items on the source environment after deleting the backoffice language(s), the newly created Development environment/child project should have no issues whatsoever.
+
+(*Method 2*)  - if you have already created the new instance and do not wish to re-create it, you could instead navigate to site/wwwroot/data/revision folder via [KUDU tools](../../../Set-up/Power-Tools/index), find the dictionary items, and edit the UDA files directly.
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/Deployments/Language-Mismatch/index.md:43]:
```suggestion
*Method 2*  - if you have already created the new instance and do not wish to re-create it, you could instead follow this flow: Navigate to `site/wwwroot/data/revision` folder via [KUDU tools](../../../Set-up/Power-Tools/index), find the dictionary items, and then edit the UDA files directly.
```

**Pull request merged**